### PR TITLE
image from rois

### PR DESCRIPTION
**Pull request by jburel**
Fix script from Alex
Move the script to our example repo so we can link from https://www.openmicroscopy.org/site/community/scripts

To test:

 * Upload the script as an admin using for example insight
 * Select an image with rectangular rois
 * Run the script

cc @hflynn @pwalczysko

### CODE REVIEW

File: processing_scripts/New_Images_From_ROIs.py [hunk 1]
```diff
@@ -4,6 +4,9 @@
 images with the regions within the ROIs, and saves them back to the server.
 
 This script is adapted from [SCRIPTS]/omero/util_scripts/Images_From_ROIs.py
+This is an adapted version of Alex Herbert's script so it can be used
```
Comment (hflynn) [inline processing_scripts/New_Images_From_ROIs.py]: I'd move this new info to below 'The script has been modified to allow...' info - it's a bit confusing to put it here after the mention of the script Alex originally adapted it from.

**Comment by hflynn** [comment]
👍

**Comment by jburel** [comment]
The remaining flake8 issue should be fixed when https://github.com/openmicroscopy/omero-example-scripts/pull/4 is merged

**Comment by will-moore** [comment]
Not tested all different options/behaviours but seems to work OK.
Good to merge.

**Pull request merged**